docker/scripts/cts/README

```diff
@@ -0,0 +1,70 @@
+The scripts in this folder can be used to build a docker image for Java EE CTS 
+and run the tests in a docker container, using the base image centos:7.
+
+How to build the image ?
+-------------------------
+Run build.sh from docker/scripts/cts to build the docker image will contain 
+JDK 8 (jdk-8u171) and Apache Ant 1.9.7. You can edit the build.sh to set the 
+URL to download the JDK8, the script expects you to supply a tar.gz file 
+accessible using http.
+
+By default, the image will contain the Java Web Services Developer Pack (JWSDP)
+1.3 and JDK 1.5 required to run JWSDP. You can provide the JWSDP
+bundle and JDK by supplying the values for JWSDP_BUNDLE and JDK_FOR_JWSDP in
+build.sh.
+
+Setup JavaMail Server
+-------------------------
+There are tests in tests/samples and tests/javamail which require a JavaMail
+Server. The docker scripts in the current form expect you to run the Apache 
+James image, in the same machine, with IMAP port 143 and SMTP port 25.
+
+For example, you can run the command 
+     docker run -p "45:25" -p "243:143" linagora/james-jpa-sample:3.0.1
+
+TODO : Provide the support to run the CTS with any given Mail Server	 
+
+How to run the tests from CTS in a container ?
+---------------------------------------------
+You can run the docker image in multiple ways, by passing one or more arguments
+it accepts as environment variables. Here are sample docker run commands - 
+
+1. Run the image to download the Java EE CTS bundle from an internal location
```

Comment:
Probably should replace Java EE with Jakarta EE.

Reply:
Hi Scott - all the files under docker/scripts does not affect the job as they require changes to be used for execution. They are available to the community, for future reference and so that build and run can be done in a local machine.